src/circuit/circuit_frame.rs

```diff
@@ -1250,6 +1267,130 @@ fn reduce_sym<F: LurkField, CS: ConstraintSystem<F>>(
     Ok((output_expr, output_env, output_cont, apply_cont_num))
 }
 
+#[derive(Clone, Copy)]
+pub struct Pointers<F: LurkField> {
```

Comment:
Let's leave it for later. Just a note that `KeywordPointers` is definitely a bad choice, since Lurk has a keyword type (`Key`) that is distinct from this usage. The important point is really that these are constants that will always have the same `ScalarPtr` component values for any program. We might want to also include `ScalarContPtrs`. Just some food for thought about best direction to push things.

Reply:
Happy to leave the name yak-shaving for later. 
> these are constants that will always have the same ScalarPtr component values for any program. 

That would do great as a rustdoc comment!